azure/common_columns.go

```diff
@@ -66,7 +66,7 @@ func getEnvironmentName(ctx context.Context, d *plugin.QueryData, h *plugin.Hydr
 	}
 
 	// cache environment name for the session
-	d.ConnectionManager.Cache.Set(cacheKey, session.EnvironmentName)
+	d.ConnectionManager.Cache.Set(cacheKey, session.CloudEnvironment)
```

Comment:
We updated references here, but I think we should update all other references starting on line 55 too